docs/notebooks/tutorials/EffectiveRipple.ipynb

```diff
@@ -0,0 +1,480 @@
+{
```

Comment:
<p>should include a bit of intro/explanation before getting into the code. What is effective ripple? why do we care about it? How do we optimize it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/1290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>